frontend/src/app/components/wp-card-view/wp-single-card/wp-single-card.component.html

```diff
@@ -1,69 +1,48 @@
-<div class="wp-card"
-     [ngClass]="cardClasses()">
+<div class="wp-card" [ngClass]="cardClasses()">
 
-  <div class="wp-card--highlighting"
-       [ngClass]="cardHighlightingClass(workPackage)">
+  <div class="wp-card--highlighting" [ngClass]="cardHighlightingClass(workPackage)">
   </div>
 
   <div class="wp-card--inline-buttons hidden-for-mobile">
-    <a class="wp-card--inline-cancel-button -no-decoration"
-       *ngIf="workPackage.isNew || showRemoveButton"
-       [ngClass]="{ '-show': workPackage.isNew }"
-       [title]="text.removeCard"
-       (accessibleClick)="onRemoved(workPackage)">
+    <a class="wp-card--inline-cancel-button -no-decoration" *ngIf="workPackage.isNew || showRemoveButton"
```

Comment:
Please undo all the changes where you only removed the line breaks. Writing all attributes into one line is really hard to read. You have to check very closely which attributes are set and which not.